portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,49 +19,63 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
 import java.util.ArrayList;
+import java.util.Stack;
 
 import org.json.simple.JSONObject;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/comment")
 public class DataServlet extends HttpServlet {
-  ArrayList<String> allComments = new ArrayList<>();
-
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    if (allComments.size() == 0) {
-        allComments.add("Welcome to my mailbox :)");
-        allComments.add("Hi, I am Yi Lyu. My foreign name is Isabella :D");
-        allComments.add("I am looking forward to know u.");
-        allComments.add("You can contact me with the info in the left side bar!");
-    }
-    
-    String json = convertToJson();
+    String json = convertAllCommentsToJson();
     response.setContentType("application/json");
     response.getWriter().println(json);
   }
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    if (allComments.size() == 0) {
-        allComments.add("Welcome to my mailbox :)");
-        allComments.add("Hi, I am Yi Lyu. My foreign name is Isabella :D");
-        allComments.add("I am looking forward to know u.");
-        allComments.add("You can contact me with the info in the left side bar!");
-    }
-    
     String comment = request.getParameter("newComment");
-    System.out.println(allComments.size());
-    allComments.add(comment);
+
+    // Write comment to comment.txt file
+    FileWriter fw = new FileWriter("/home/yilyu/software-product-sprint/portfolio/src/main/java/com/google/sps/servlets/comment.txt", true);
```

Comment:
I thought you are supposed to use blobstore or other storage solutions, instead of saving to file?